### ISSUE TITLE
Sets subcondition?

### ISSUE THREAD

**Issue by paramecie** [rgriebl/brickstore#119]
Subcondition is mandatory (AFAIK) on BrickLink's add item page for a Set, as a "C", "I", or "S".

It's still optional on mass upload, but until when?

And the problem is that I upload with no sub-condition (using BrickStore), then sometimes an order is cancelled and BrickLink **forces** me to set "S" for example when I put the order back in shop during the cancellation.

So, then I've a Duplicate, my uploaded Sets with no sub-condition, and 1 identical Set written "Complete" (or "Sealed").
It happens to me from time to time, especially dealing with the Collectible Minifigures Series...

**In short, maybe it'd be interesting to have a subcondition for sets?**

(which may be "" blank by default, and could be reset to blank if you wish to keep your habits)

PS: as "New" or "Used", the default subcondition could be also set up in Options/Defaults ;-)

_And please don't tell me it's already exisiting on BrickStore and that I didn't see it - I'll hate you ;-)_

**Comment by rgriebl** [comment]
It is already there. You have to use the menu to set it though (context or edit). Just noticed that it displays weirdly in the condition column ... need to fix that.

**Comment by paramecie** [comment]
Yes/no - I meant in the Add Item window - sorry.

Right now you've to add a Set, then use context or menu to set Sealed or Complete.

And a default option (like used/New) would be helpful.

**Comment by rgriebl** [comment]
Fixed in 1ab22f7

**Comment by paramecie** [comment]
Thank you million much.
